### ISSUE TITLE
Item and optional_item arguments not working for ukppd_get()

### ISSUE THREAD

**Issue by zyt** [kvasilopoulos/uklr#1]
I encountered the following problem. When I try to specify that I want to extract additional data by using the **item** or **optional_item** arguments
```r
ukppd_get(postcode="PL6 8RU", optional_item = "street", start_date = "2015-01-01")
```
I get the following error:
```r
Error in sparql(query = req, ...) : Bad Request (HTTP 400).
```

I get the same error if I try to pick any of the extra items from **ukppd_avail_items()** or **ukppd_avail_items()**

**Comment by kvasilopoulos** [comment]
I am aware of this problem. This may take some time since some items and optional items return the whole URI. You can try the development version, but it still has some rough edges.

```r
remotes::install_github("kvasilopoulos/uklr")
library(uklr)
ukppd_get(postcode="PL6 8RU", item = "newBuild", optional_item = "street")
#> # A tibble: 32 x 6
#>    postcode amount date       category                    newBuild street       
#>  * <chr>     <dbl> <date>     <chr>                       <chr>    <chr>        
#>  1 PL6 8RU  227000 2007-06-29 Standard price paid transa~ false    PATTINSON DR~
#>  2 PL6 8RU  149950 2002-11-08 Standard price paid transa~ false    PATTINSON DR~
#>  3 PL6 8RU  220000 2005-06-29 Standard price paid transa~ false    PATTINSON DR~
#>  4 PL6 8RU  195000 2004-06-17 Standard price paid transa~ false    PATTINSON DR~
#>  5 PL6 8RU  172000 2008-11-14 Standard price paid transa~ false    PATTINSON DR~
#>  6 PL6 8RU  180000 2014-06-02 Standard price paid transa~ false    PATTINSON DR~
#>  7 PL6 8RU   58000 2000-05-19 Standard price paid transa~ false    PATTINSON DR~
#>  8 PL6 8RU  179000 2011-04-20 Standard price paid transa~ false    PATTINSON DR~
#>  9 PL6 8RU  192500 2011-07-27 Standard price paid transa~ false    PATTINSON DR~
#> 10 PL6 8RU  202500 2015-06-29 Standard price paid transa~ false    PATTINSON DR~
#> # ... with 22 more rows
```

**Comment by kvasilopoulos** [comment]
Resolved in uklr v1.0.1 https://github.com/kvasilopoulos/uklr/commit/e2bb183ea62be8252c6c5c064c6f427faf9de294
